flexget/plugins/input/anilist.py

```diff
@@ -114,6 +114,8 @@ def on_task_input(self, task, config):
                 list_response = list_response.json()['data']
                 logger.debug('JSON output: {}', list_response)
                 for list_status in list_response['collection']['statuses']:
+                    if not list_status.get('status'):
```

Comment:
does it makes sense for a list not to have a status?

Reply:
It does. 'Native' lists are based on status, Custom lists only have a name (I just don't fetch that field in the query since it's not used by the plugin).
Example result where `Winter 2020` is a custom list (added the list `name` field to the query, stripped all but one title field and limited to 1 anime entry for this):
```
{
  "data": {
    "collection": {
      "hasNextChunk": true,
      "statuses": [
        {
          "status": "CURRENT",
          "name": "Watching",
          "list": [
            {
              "anime": {
                "status": "RELEASING",
                "title": {
                  "english": "My Hero Academia Season 4"
                }
              }
            }
          ]
        },
        {
          "status": null,
          "name": "Winter 2020",
          "list": [
            {
              "anime": {
                "status": "RELEASING",
                "title": {
                  "english": "My Hero Academia Season 4"
                }
              }
            }
          ]
        }
      ]
    }
  }
}
```